tests/model/range.js

```diff
@@ -1056,6 +1059,22 @@ describe( 'Range', () => {
 				expect( transformed[ 0 ].start.path ).to.deep.equal( [ 0, 0, 1 ] );
 				expect( transformed[ 0 ].end.path ).to.deep.equal( [ 0, 1, 1 ] );
 			} );
+
+			// #1132
+			it( 'merge delta has move operation that does not start from offset 0', () => {
```

Comment:
Such tests without initial comments like these in lines 1048-1051 are completely unreadable for me (at least ;P)